src/utils/line.ts

```diff
@@ -349,19 +514,16 @@ export const formatLinePageData = (
     ...metadata,
     // debt data
     ...credit,
-    creditEvents: [], // @ TODO add to formatSecuredLineData
+    creditEvents,
+    collateralEvents: collateralEvents,
```

Comment:
and break collateralEvents into `escrowEvents` and `spigotEvents`

Reply:
I'm only including `creditEvents` at the top level now. Is there a desire to have both `escrowEvents` and `spigotEvents` at the top level? seems unnecessary given that you can access them through `escrow.events` and `spigot.events`.